extension/intents/timer/timer.toml

```diff
@@ -3,6 +3,7 @@ description = "Set timer"
 match = """
     (set | open | show | start | create | make |) (a |) (timer | countdown) (for |) [time]
     (set | start | create | make |) (for |) (alarm ) [time] (from now |)
+    [time] minutes [sufixQuery=minutes]
```

Comment:
Cool, but we could match some more patterns here:

```
[time] minute{s} (timer |) (from now |)
[time] hour{s} (timer |) (from now |)
[time] second{s} (timer |) (from now |)
```

It will require just a bit more code to support:

```
[minutes] minute{s} (and |) [seconds] second{s} (timer |) (from now |)
[hours] hour{s} (and |) [minutes] minutes{s} (timer |) (from now |)
```

Reply:
@ianb I have added the following pattern as told by you:
[time] minute{s} (timer |) (from now |)
[time] hour{s} (timer |) (from now |)
[time] second{s} (timer |) (from now |)

Regarding the following pattern:
[minutes] minute{s} (and |) [seconds] second{s} (timer |) (from now |)
[hours] hour{s} (and |) [minutes] minutes{s} (timer |) (from now |)

The above pattern is already getting executed without any changes, Please see below example:
![ezgif com-video-to-gif (6)](https://user-images.githubusercontent.com/65556960/88453978-8ecf1980-ce89-11ea-846e-f23de3c4b2bd.gif)

Thanks.